packages/ui-library/src/components/select-field/stories.js

```diff
@@ -83,6 +82,15 @@ WithLabelSuffix.args = {
 	value: "3",
 	label: "Select field with a label suffix",
 	labelSuffix: <Badge className="yst-ml-1.5" size="small">Beta</Badge>,
+	children: options.map( option => <SelectField.Option key={ option.value } { ...option } /> ),
+};
+
+export const OptionsProp = Template.bind( {} );
```

Comment:
This might not be needed in the SelectField docs if it lives in the Select docs?

Reply:
After we spoke, decided to leave it after all.